buildenv/docker/mkdocker.sh

```diff
@@ -532,32 +543,33 @@ prepare_user() {
 create_user() {
   echo ""
   echo "# Add user home and copy authorized_keys and known_hosts."
-  echo "RUN useradd -ms /bin/bash $user \\"
+  echo "RUN useradd -ms /bin/bash --uid $userid $user \\"
   echo " && mkdir /home/$user/.ssh \\"
   echo " && chmod 700 /home/$user/.ssh"
   echo "COPY authorized_keys known_hosts /home/$user/.ssh/"
-  echo "RUN chown -R $user:$user /home/$user \\"
-  echo " && chmod 600 /home/$user/.ssh/authorized_keys /home/$user/.ssh/known_hosts"
+  echo "RUN chmod 600 /home/$user/.ssh/authorized_keys /home/$user/.ssh/known_hosts \\"
+  echo " && echo \"PS1=\\\"\\u:DOCKER-CONTAINER $ \\\"\" > /home/$user/.bashrc \\"
+  echo " && echo \"ulimit -u 32000\" >> /home/$user/.bashrc \\"
+  echo " && echo \"ulimit -n 2048\" >> /home/$user/.bashrc"
+}
+
+adjust_user_directory_perms() {
+  echo "RUN chown -R $user:$user /home/$user"
 }
 
 install_freemarker() {
   echo ""
   local freemarker_version=2.3.8
-  echo "# Download and extract freemarker.jar to /root/freemarker.jar."
-  echo "RUN mkdir -p /root \\"
-  echo " && cd /root \\"
+  echo "# Download and extract freemarker.jar to /home/$user/freemarker.jar."
```

Comment:
Does `buildenv/ansible/roles/openj9-from-sources/tasks/main.yml` need to change too?

Reply:
I don't see how that is related to this bit of code?  But I suppose that can be modified as well.  The docker files that were replaced with this script were originally written for the build system to use which uses a basic user and not root user.  The user specific changes from /root -> /home/$user/ will mean a minor change to devs that use these yes, but if they are running as root they should have full access /home/$user/.

For freemarker.jar...is this actually a requirement for x/p/z linux platforms any more?

Reply:
x/p/z Linux all use cmake by default. It may be time to remove freemarker from the script.
Opinions, @pshipton?

Reply:
How about we comment it out, so it's not installed by default, but a user can uncomment if needed.

Reply:
If you mean hide the call to `install_freemarker` like this, I agree:
```
# install_freemarker
```

Reply:
Yes, that is what I mean.

Reply:
Commented out in my latest push.